src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -60,7 +60,7 @@
     r'.*(AddressSanitizer).*memcpy-param-overlap'
     r'[^\[]*([\[].*[)])')
 ASAN_REGEX = re.compile(
-    r'.*ERROR: (HWAddressSanitizer|AddressSanitizer)[: ]*[ ]*([^(:]+)')
+    r'.*ERROR: (HWAddressSanitizer|AddressSanitizer)[: ]*[ ]*([^(:;]+)')
```

Comment:
I've checked the code, looks like `allocation-tail-overwritten` is the only signature that has `;`. Should as ask @eugenis to change it, if possible?

Reply:
It will take a while for the change to make it in build system, and this seems simple enough to add as a char.

Reply:
yes, definitely! I meant for the long term we can ask @eugenis to get rid of it together with fixing the typo